perfmetrics/scripts/continuous_test/gcp_ubuntu/build.sh

```diff
@@ -30,7 +30,7 @@ cd "./perfmetrics/scripts/"
 echo Mounting gcs bucket
 mkdir -p gcs
 LOG_FILE=log-$(date '+%Y-%m-%d').txt
-GCSFUSE_FLAGS="--implicit-dirs --max-conns-per-host 100 --experimental-enable-storage-client-library=true --disable-http2 --debug_fuse --debug_gcs --log-file $LOG_FILE --log-format \"text\" --stackdriver-export-interval=30s"
+GCSFUSE_FLAGS="--implicit-dirs --max-conns-per-host 100 --experimental-enable-storage-client-library=true --debug_fuse --debug_gcs --log-file $LOG_FILE --log-format \"text\" --stackdriver-export-interval=30s"
```

Comment:
what is the default value of the flag if nothing is passed?

Reply:
the default value is http1: https://github.com/GoogleCloudPlatform/gcsfuse/blob/1cd60205d80ce4ec286aeda540b7af7c80d302e1/flags.go#L248. 
gcsfuse --help will show that default is http1.